### PR TITLE
Add more content to the prototyping workshop

### PR DESCRIPTION
**Pull request by kaelig**

### CODE REVIEW

**Comment by kaelig** [comment]
cc @rajdeepd

File: content/workshop/rapid-prototyping-with-slds/index.md [hunk 1]
```diff
@@ -104,19 +130,48 @@ Feel free to add more components to the page and go crazy! You'll see the result
...
 {{% fluid_img "images/edit-in-browser.gif" %}}
+
+#### Editing styles in the browser
+
 {{% fluid_img "images/edit-scss-in-browser.gif" %}}
 
 ## From Sketch to the browser (section title TBD)
 
-TODO: instructions to copy - paste some components into a view.
+Let's do some copy and pasting! Errr… some coding!
```
Comment (mcsuth) [inline content/workshop/rapid-prototyping-with-slds/index.md:157]: :-D

File: content/workshop/rapid-prototyping-with-slds/index.md [hunk 2]
```diff
@@ -104,19 +130,48 @@ Feel free to add more components to the page and go crazy! You'll see the result
...
+
+In the `src/views` directory, there's a file called **`workshop.html`**. That's where we're going to make the magic happen.
+
+TODO: More instructions to copy - paste some components into a view.
```
Comment (mcsuth) [inline content/workshop/rapid-prototyping-with-slds/index.md:161]: Maybe we don't need more instructions if we have comments in the `workshop.html` file? Just a thought. :-)

Comment (kaelig) [reply content/workshop/rapid-prototyping-with-slds/index.md:161]: That'd be nice. Let's see how the dry run goes without it!

**Pull request merged**